TODO : Due to bug described here: https://github.com/pymc-devs/pymc3/issues/3122 , need > 1 missing data. Make sure datasets used in Blang and PyMC3 are matching.
